ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -114,38 +115,53 @@ export default function TransactionList({
     [],
   );
 
-  const pendingLength = pendingTransactions.length;
-
   return (
     <div className="transaction-list">
       <div className="transaction-list__transactions">
-        {pendingLength > 0 && (
+        {pendingTransactions.length > 0 && (
           <div className="transaction-list__pending-transactions">
             <div className="transaction-list__header">
               {`${t('queue')} (${pendingTransactions.length})`}
             </div>
-            {pendingTransactions.map((transactionGroup, index) => (
-              <TransactionListItem
-                isEarliestNonce={index === 0}
-                transactionGroup={transactionGroup}
-                key={`${transactionGroup.nonce}:${index}`}
-              />
-            ))}
+            {pendingTransactions.map((transactionGroup, index) =>
+              transactionGroup.initialTransaction.transactionType ===
+              'smart' ? (
```

Comment:
Let's create a constant for `'smart'`. It can be added to the `TRANSACTION_TYPES` object of `shared/constants/transaction.js`